### PR TITLE
Closing 9515

### PR DESCRIPTION
**Pull request by shubham11941140**
Issue: #9515

### CODE REVIEW

**Comment by timurt** [comment]
Hi @shubham11941140,

Please provide screenshots and cmd line execution results for `MINUS` token (you can use https://github.com/checkstyle/checkstyle/pull/9378 as an example)

File: src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java [hunk 1]
```diff
@@ -2331,6 +2331,21 @@
     /**
      * The {@code |} (bitwise OR) operator.
      *
+     * <p>For example:</p>
+     * <pre>
+     * a = a | b;
```
Comment (timurt) [inline src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java:2336]: scope of this issue is only `MINUS` token, you do not need to do other tokens

**Review by timurt — CHANGES_REQUESTED** [review]

File: src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java (hunk 1 above)
Comment (shubham11941140) [reply src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java:2336]: The MINUS part is right yet some checks are failing, mvn verify says build successful, what to do?

Comment (shubham11941140) [reply src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java:2336]: There is only one commit, I have rebased and squashed the commits but yet it fails?
How to solve that, I have no clue and the code is perfect?

Comment (timurt) [reply src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java:2336]: please check your `shubham11941140:master` it says `This branch is 1 commit ahead, 41 commits behind checkstyle:master.`

fetch latest updates from checkstyle master and rebase on it

**Comment by romani** [comment]
@shubham11941140 , refrenced issue is invalid, please reference correct issue.

**Comment by timurt** [comment]
@shubham11941140 as romain said, please correct issue reference, 

also please put screenshots and cmd outputs in PR description like [here](https://github.com/checkstyle/checkstyle/pull/9378)

Also your PR contains examples for two token types, but you are supposed to write example only for one

**Comment by romani** [comment]
"minus" already merged

**Pull request closed without merging**